ALI/ALI.py

```diff
@@ -8,6 +8,7 @@
 from slicer.util import VTKObservationMixin
 import webbrowser
 import textwrap
+import pkg_resources
```

Comment:
I guess I overlooked this before since `pkg_resources` was not imported till now. `pkg_resources` is deprecated and should not be used. It looks like the only usage is in `check_lib_installed`. You could perform the same task with `importlib.metadata`, something like:

```py
import importlib.metadata

...

installed_version = importlib.metadata.version(lib_name)
```

---

I tried to make the comment on `check_lib_installed`, but that made it look like the request for changes applied to `import sys` which is wrong. Then tried to delete the wrongly placed comment and accidentally deleted the replacement. :roll_eyes: Sorry for the noise.

Reply:
Don't worry it's ok, thank you for the feedback !

I made the change, is it good now ?